backends/p4tools/modules/testgen/targets/bmv2/test/BMV2STFXfail.cmake

```diff
@@ -235,6 +235,12 @@ p4tools_add_xfail_reason(
   issue584-1-bmv2.p4
 )
 
+p4tools_add_xfail_reason(
+  "testgen-p4c-bmv2-stf"
+  "is not a constant"
```

Comment:
Nitpick: This could be more precise with an comment on the actual error message. 
Like
https://github.com/fruffy/flay/blob/master/targets/v1model/test/BMv2V1ModelXfail.cmake#L8
for example.